assets/stylesheets/modules/ai-bot-conversations/common.scss

```diff
@@ -33,10 +33,7 @@ body.has-ai-conversations-sidebar {
           opacity: 0.8;
           font-weight: 700;
           margin-top: 1em;
-
-          &[data-link-name="date-heading-last-7-days"] {
-            margin-top: 0;
-          }
```

Comment:
This was removing margin on the first grouping date label in the sidebar... we can just have margin on them all to keep consistency.